add generic guidelines #17 (#22)
MadhavBahlMD/my-open-profile: README.md

Do ⭐ this repo.

> contributing

Reviewer: ```suggestion
## Contributing
```

> - See [Contributing Guidelines](https://github.com/MadhavBahlMD/my-open-profile/blob/master/CONTRIBUTING.md) for great contribution.

Reviewer: ```suggestion
- Refer to the  [Contributing Guidelines] document (https://github.com/MadhavBahlMD/my-open-profile/blob/master/CONTRIBUTING.md) for tips to make a great contribution.
```

> - Refer to the  [Contributing Guidelines] document (https://github.com/MadhavBahlMD/my-open-profile/blob/master/CONTRIBUTING.md) for tips to make a great contribution.

Reviewer 2: ```suggestion
- Refer to the  [Contributing Guidelines Document](https://github.com/MadhavBahlMD/my-open-profile/blob/master/CONTRIBUTING.md) for tips to make a contribution.
```
